critiquebrainz/data/model/user_test.py

```diff
@@ -148,21 +148,21 @@ def test_reviews_property(self):
         self.assertEqual(len(user.reviews), 0)
 
     def test_avatar_property(self):
-        user = User(display_name=u'Tester', email=u"example@example.org")
+        user = User(display_name=u'Tester', email=u"example@example.org", id=u"ea1c2bf3-f246-d47c-f6a4-e5f6e7b8d9e0")
         db.session.add(user)
         db.session.commit()
 
         # By default show_gravatar attribute should be set to False
         self.assertFalse(user.show_gravatar)
         # so avatar property returns generic avatar
-        self.assertEqual(user.avatar, "https://gravatar.com/avatar/placeholder?d=mm")
+        self.assertEqual(user.avatar, "https://gravatar.com/avatar/476c357ea2c081b90bf2a6659d0fbd26?d=identicon")
 
         # Let's allow to show avatar of this user.
         user.show_gravatar = True
         db.session.commit()
 
         self.assertTrue(user.show_gravatar)
-        self.assertEqual(user.avatar, "https://gravatar.com/avatar/f72c502e0d657f363b5f2dc79dd8ceea?d=mm&r=pg")
+        self.assertEqual(user.avatar, "https://gravatar.com/avatar/f72c502e0d657f363b5f2dc79dd8ceea?d=identicon&r=pg")
```

Comment:
What we could do is perhaps generate random email id and then assign that to user and check if the correct hash for it is there in the gravatar url